src/result_gen.py

```diff
@@ -0,0 +1,563 @@
+import os
+import sys
+import json
+import shutil
+import random
+import numpy as np
+from bunch import Bunch
+
+import config
+import utils.inference as inference
+import utils.dbutils as dbutils
+import utils.blob_access as blob_access
+import utils.rgutils as rgutils
+import utils.preprocessing as preprocessing
+
+
+def get_stats(predictions):
+    res = Bunch()
+    res.mean = str(np.mean(predictions))
+    res.min = str(np.min(predictions))
+    res.max = str(np.max(predictions))
+    res.std = str(np.std(predictions))
+    return res
+
+
+def get_artifact_result(artifacts, predictions):
+    results = []
+    for artifact, prediction in zip(artifacts, predictions):
+        artifact_result = Bunch()
+        artifact_result.artifact_id = artifact[0]
+        artifact_result.path = '/'.join(artifact[3].split('/')[4:])
+        artifact_result.prediction = str(prediction[0])
+        results.append(artifact_result)
+    return results
+
+
+def move_processed_data(json_path, table, state):
+    # destination_folder1 = destination_folder + "{0}_{1}".format(state, table)
+    destination_folder = '/'.join(json_path.split('/')[:-3])
+    destination_folder = destination_folder + \
+        "/" + "{0}_{1}".format(state, table)
+    if not os.path.exists(destination_folder):
+        os.makedirs(destination_folder)
+    try:
+        filename = json_path.split('/')[-1]
+        shutil.move(json_path, os.path.join(destination_folder, filename))
+    except Exception as error:
+        print(error)
+
+
+def get_file_name(destination_folder, model_id, qr_code, environment):
+    filename = '{0}-{1}-{2}-{3}.json'.format(
+        environment, model_id, qr_code, random.randint(10000, 99999))
+    folder = '{0}/{1}/'.format(destination_folder, model_id)
+    return filename, folder
+
+
+def extract_status_code_one(path):
+    '''
+    Extract the status code from the artifact path
+    '''
+    status_code = path.split('/')[-1].split('_')[-2]
+    return status_code
+
+
+def extract_status_code_two(path):
+    '''
+    Extract the status code from the artifact path
+    '''
+    status_code = path.split('/')[-1].split('_')[-3]
+    return status_code
+
+
+def check_status_code(path, extract_status_code, status_list):
+    '''
+    Check the artifact belong to one of the status code in status list
+    '''
+
+    if extract_status_code(path) in status_list:
+        return True
+    else:
+        return False
```

Comment:
Personally I prefer to shorten code if it is still as readable afterwards

```suggestion
    return extract_status_code(path) in status_list:
```